data/hai/hai news.txt

```diff
@@ -114,6 +190,24 @@ news "concerned merchant in Hai space"
 		word
 			`"`
 
+news "concerned hai in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Hai merchant"
+	message
+		word
+			`"`
+		word
+			"Be careful should you choose to fly up north. Too many human captains who don't know any better become victims to those Hai seeking great trouble."
+			"It would be smart to follow trade routes that only visit Hai and human space. One would not wish to experience the business of our northern brethren."
+			"Seeing so many human ships come by with such extensive weapons feels very reminiscent of past times, unfortunately."
+			"Rations are very often sent to those starving to the north of here, but such actions are very often not met with similar kindness. Take such a task very carefully."
+			"I fear what may happen to those who attempt to disrupt the Quarg to the south. They do stay out of the way, but do become retaliatory if initiated."
```

Comment:
The Hai probably wouldn't have fired on the Quarg in nearly 180,000 years. I wouldn't say this one is particularly sensible to include.